tests/api.c

```diff
@@ -33597,8 +33597,240 @@ static int test_wc_RNG_GenerateBlock(void)
+    }
+
+    printf(resultFmt, ret == 0 ? passed : failed);
+#endif
+    return ret;
+
+}/* End test_get_rand_digit*/
+/*
+ * Testing get_digit_count
+ */
+static int test_get_digit_count (void)
+{
+    int ret = 0;
+#if !defined(WOLFSSL_SP_MATH) && defined(WOLFSSL_PUBLIC_MP)
+    mp_int a;
+
+    printf(testingFmt, "get_digit_count()");
+
+    if (mp_init(&a) != MP_OKAY) {
+        ret = -1;
+    }
+    if (ret == 0) {
+        ret = get_digit_count(NULL);
+    }
+    if (ret == 0) {
+        ret = get_digit_count(&a);
+    }
+
+    printf(resultFmt, ret == 0 ? passed : failed);
+    mp_clear(&a);
+#endif
+    return ret;
+
+}/* End test_get_digit_count*/
+/*
+ * Testing mp_cond_copy
+ */
+static int test_mp_cond_copy (void)
+{
+    int     ret = 0;
+#if defined(WOLFSSL_PUBLIC_MP)    
+    mp_int  a;
+    mp_int  b;
+    int     copy = 0;
+
+    printf(testingFmt, "mp_cond_copy()");
+
+    if (mp_init(&a) != MP_OKAY) {
+        ret = -1;
+    }
+    if (ret == 0) {
+        if (mp_init(&b) != MP_OKAY) {
+            ret = -1;
+        }
+    }
+    if (ret == 0) {
+        ret = mp_cond_copy(NULL, copy, NULL);
+        if (ret == BAD_FUNC_ARG) {
+            ret = 0;
+        }
+    }
+    if (ret == 0) {
+        ret = mp_cond_copy(NULL, copy, &b);
+        if (ret == BAD_FUNC_ARG) {
+            ret = 0;
+        }
+    }
+    if (ret == 0) {
+        ret = mp_cond_copy(&a, copy, NULL);
+        if (ret == BAD_FUNC_ARG) {
+            ret = 0;
+        }
+    }
+    if (ret == 0) {
+        ret = mp_cond_copy(&a, copy, &b);
+    }
+
+    printf(resultFmt, ret == 0 ? passed : failed);
+    mp_clear(&a);
+    mp_clear(&b);
+#endif    
+    return ret;
+
+}/* End test_mp_cond_copy*/
+/*
+ * Testing mp_rand
+ */
+static int test_mp_rand (void)
+{
+    int ret = 0;
+#if defined(WC_RSA_BLINDING) && defined(WOLFSSL_PUBLIC_MP) 
+    mp_int  a;
+    int     digits = 1; /*Setting equal to 0 causes a segmentation fault, line 183*/
+    WC_RNG  rng;
+
+    printf(testingFmt, "mp_rand()");
+
+    if (mp_init(&a) != MP_OKAY) {
+        ret = -1;
+    }
+    if (ret == 0) {
+        ret = wc_InitRng(&rng);
+    }
+
+    if (ret == 0) {
+        ret = mp_rand(&a, digits, NULL);
+        if (ret == MISSING_RNG_E) {
+            ret = 0;
+        }
+    }
+    if (ret == 0) {
+        ret = mp_rand(NULL, digits, &rng);
+        if (ret == BAD_FUNC_ARG) {
+            ret = 0;
+        }
+    }
+    if (ret == 0) {
+        ret = mp_rand(&a, digits, &rng);
+    }
+
+    printf(resultFmt, ret == 0 ? passed : failed);
+    mp_clear(&a);
+    wc_FreeRng(&rng);
+#endif
+    return ret;
+}/* End test_mp_rand*/
+/*
+ * Testing mp_rand
+ */
+static int test_get_digit (void)
+{
+    int     ret = 0;
+#if defined(WOLFSSL_PUBLIC_MP)     
+    mp_int  a;
+    int     n = 0;
+
+    printf(testingFmt, "get_digit()");
+
+    if (mp_init(&a) != MP_OKAY) {
+        ret = -1;
+    }
+    if (ret == 0) {
+        if (get_digit(NULL, n) != 0) { /* Should not hit this*/
```

Comment:
Minor formatting request - can you equalize spacing on both sides of this comment?  For example "/* Should not hit this*/" to "/* Should not hit this */".  If this takes a line over 80 characters, please move comment above get_digit() line instead.  Also apply to other areas in this PR that are similar.